pslab/external/MLX90614.py

```diff
@@ -1,46 +1,40 @@
-from __future__ import print_function
+from pslab.bus import I2CSlave
 
-
-def connect(route, **args):
-    return MLX90614(route, **args)
-
-
-class MLX90614():
+class MLX90614(I2CSlave):
+    _ADDRESS = 0x5A
+    _OBJADDR = 0x07
+    _AMBADDR = 0x06
     NUMPLOTS = 1
```

Comment:
Looks like some class-level vars are unused (`NUMPLOTS`, `PLOTNAMES`, `name`). Let's remove them.

Reply:
Yes, that is also what I see in other sensors here.
For the other PR, I dropped a note on that:
https://github.com/fossasia/pslab-python/pull/192#issue-668877140

It was a previous attempt to abstract the sensors in a way, and at the same time, it is tightly coupling them with the UI. I am seeing a very similar thing in sigrok. Let's discuss this in the nxt meeting a bit further in depth please, because that information is needed in some form.

Reply:
I'm imagining some kind of `SensorMetaDataMixin` class could be used for this.